testcases/OpTestLPM.py

```diff
@@ -64,6 +64,10 @@ def setUp(self, remote_hmc=None):
         self.options = self.conf.args.options if 'options' in self.conf.args else None
         self.lpm_timeout = int(self.conf.args.lpm_timeout) if 'lpm_timeout' in self.conf.args else 300
         self.util = OpTestUtil(OpTestConfiguration.conf)
+        self.os_file_logs = self.conf.args.os_file_logs.split(",") if 'os_file_logs' in self.conf.args else []
+        self.os_cmd_logs = self.conf.args.os_cmd_logs.split(",") if 'os_cmd_logs' in self.conf.args else []
```

Comment:
if no input logs is given,than is it still collect the default logs rite ?

Reply:
Yeah, it does